### PR TITLE
Make credentials structure consistent

### PR DESCRIPTION
**Pull request by nanobash**
## Links

[Preview](https://preview.redoc.ly/all-rebilly/sort-tags-asc-ord/tag/Service-credentials/operation/GetServiceCredentialCollection)

## Checklist

- [x] Make credentials structure consistent by removing folders containing only one file
- [x] Remove not needed block scalar
- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Review by noon-dawg — DISMISSED** [review]

**Comment by nanobash** [comment]
@Rebilly/php please review

File: openapi/components/schemas/Credentials/AvalaraCredentials/AvalaraCredential.yaml
Comment (ninjasimon) [inline openapi/components/schemas/Credentials/AvalaraCredentials/AvalaraCredential.yaml:1]: I think that this folder structure (folders with only a single file inside) was [specifically recommended against](https://github.com/Rebilly/api-definitions/pull/1102#pullrequestreview-1194997177), which is why the `AvalaraCredential` schema is in its current location.

**Review by ashkarpetin — CHANGES_REQUESTED** [review]
See https://github.com/Rebilly/api-definitions/pull/1366/files#r1174134151

File: openapi/components/requestBodies/PostServiceCredential.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ content:
         - $ref: ../schemas/Credentials/SendGridCredentials/SendGridCredential.yaml
         - $ref: ../schemas/Credentials/SESCredentials/SESCredential.yaml
         - $ref: ../schemas/Credentials/TaxJarCredentials/TaxJarCredential.yaml
-        - $ref: ../schemas/Credentials/AvalaraCredential.yaml
+        - $ref: ../schemas/Credentials/AvalaraCredentials/AvalaraCredential.yaml
```
Comment (ashkarpetin) [inline openapi/components/requestBodies/PostServiceCredential.yaml:14]: Better to do the opposite, and remove all nesting from `Credentials`

File: openapi/components/schemas/Credentials/AvalaraCredentials/AvalaraCredential.yaml
Comment (nanobash) [reply openapi/components/schemas/Credentials/AvalaraCredentials/AvalaraCredential.yaml:1]: Thanks for noting, I did not know, I'll update it.

Comment (nanobash) [reply openapi/components/schemas/Credentials/AvalaraCredentials/AvalaraCredential.yaml:1]: Thanks for noting, I did not know, I'll update it.

**Comment by nanobash** [comment]
@Rebilly/php please review

**Review by ashkarpetin — APPROVED** [review]

**Review by justnero — APPROVED** [review]

**Pull request merged**